### ISSUE TITLE
title_text_align plot property behaves strangely

### ISSUE THREAD

**Issue by aolieman** [bokeh/bokeh#2081]
The [title_text_align](http://bokeh.pydata.org/en/latest/docs/reference/models.html#bokeh.models.plots.Plot.title_text_align) property does not function properly. I've tested this in Chromium v41 and in Firefox v36.

The property is defined as `Enum(‘left’, ‘right’, ‘center’)`, but the values `'left'` and `'right'` don't produce the intended alignment. Left shifts the title a bit to the right, and right shifts the title left of center, but neither value leads to actual alignment. And how come the direction is inverted?

**Comment by bryevdv** [comment]
Can you please provide a minimum runnable example?

**Comment by aolieman** [comment]
This reproduces the issue (based on iris example):
```python
from bokeh.sampledata.iris import flowers
from bokeh.plotting import *

colormap = {'setosa': 'red', 'versicolor': 'green', 'virginica': 'blue'}
flowers['color'] = flowers['species'].map(lambda x: colormap[x])

output_file("iris.html", title="iris.py example")

p = figure(title = "Iris Morphology")
p.xaxis.axis_label = 'Petal Length'
p.yaxis.axis_label = 'Petal Width'

p.circle(flowers["petal_length"], flowers["petal_width"],
        color=flowers["color"], fill_alpha=0.2, size=10, )

# try this with either 'left' or 'right'
p.title_text_align = 'left'

show(p)
```

**Comment by bryevdv** [comment]
Thanks. There is another ticket for some other title-related improvements, we will probably tackle this at the same time.

**Comment by bryevdv** [comment]
References #149 #1449

**Comment by birdsarah** [comment]
Ref #1085

**Comment by birdsarah** [comment]
There is a new plot.title.title_align property that hopefully behaves as you might expect. If it doesn't, you now have full flexibility by adding a Label annotation.

Fixed in #4392
